Update governance doc (#2)
apollosolutions/governance-example: example_governance_doc.md

## Getting Help

To request help with the ACME graph, please reach out in #graphql in Slack. 

> The following is a proposed learning path for contributors not familiar with Apollo GraphQL

Reviewer: ```suggestion
The following is a proposed learning path for contributors not familiar with Apollo GraphQL.
```

### Learning Paths

> Apollo offers two certification courses through our Odyssey platform: [Associate](https://www.apollographql.com/tutorials/certifications/apollo-graph-associate) and [Professional](https://www.apollographql.com/tutorials/certifications/apollo-graph-professional). Contributors are recommended to complete at least the Associate certification in order to understand the basics of GraphQL and Federation.

Reviewer: ```suggestion
Apollo offers two certification courses through their Odyssey platform: [Associate](https://www.apollographql.com/tutorials/certifications/apollo-graph-associate) and [Professional](https://www.apollographql.com/tutorials/certifications/apollo-graph-professional). Contributors are recommended to complete at least the Associate certification in order to understand the basics of GraphQL and Federation.
```

> In addition to the certifications, Odyssey also offers smaller modules for focusing on certain areas:

Reviewer: ```suggestion
In addition to the certifications, Apollo also offers smaller modules for focusing on certain areas:
```

- [A bite-sized course on what federation is](https://www.apollographql.com/tutorials/introduction-to-federation)
- [Basics of GraphOS with a strong focus on Studio usage](https://www.apollographql.com/tutorials/getting-started-with-graphos)

> - [Beta course for safe API delivery into GraphOS](https://www.apollographql.com/tutorials/graphos-shipping-supergraph)

Reviewer: Not sure how I feel about noting them as beta- also feels like some doc debt we'd land in if we don't update them in lockstep.

Author: Fair enough, I'm just gonna remove those for now. It doesn't affect this doc in any meaningful way to keep them there

- [Beta mini-course on growing your supergraph](https://www.apollographql.com/tutorials/graphos-growing-supergraph)
- [Beta course around leveraging Apollo Client on a React front-end](https://www.apollographql.com/tutorials/client-side-graphql-react)

> ### Enterprise Modules:

Reviewer: ```suggestion
### Enterprise Modules

```

> Enterprise modules require ACME contributors to log into Odyssey using their SSO credentials

Reviewer: ```suggestion
Apollo's enterprise modules require ACME contributors to log into Odyssey using their SSO credentials
```

Modules might be a bit odd, but it's fine I think? Would love Docs team's thoughts here as well.

Reviewer 2: We currently use series (or learning path) > courses > lessons. A series is composed of multiple courses, a course is composed of multiple lessons.

Reviewer 3: "Enterprise courses" would be the best fit here!

Author: thanks, all! going to re-word to "enterprise courses" instead.

- [Effective graph stewardship](https://www.apollographql.com/tutorials/effective-stewardship)
- [Observability in GraphOS](https://www.apollographql.com/tutorials/supergraph-observability)

> - [Testing (unit and load testing on both the client and the server)](https://www.apollographql.com/tutorials/testing)

Reviewer: ```suggestion
- [Testing for both clients and the servers](https://www.apollographql.com/tutorials/testing)
```

> ### Additional Resources:

Reviewer: ```suggestion
### Additional Resources

```

> - A great reference for GraphQL is the official [GraphQL website](https://graphql.org/learn/)

Reviewer: ```suggestion
- Another great reference for GraphQL is the official [GraphQL website](https://graphql.org/learn/)
```

> - Teams are also encouraged to run through [Supergraph Architecture Framework (SAF) assessment](https://saf.apollographql.com/) to better understand their GraphQL posture across several pillars like Reliability, Security, Operational Excellence, Performance and Developer Experience.

Reviewer: ```suggestion
- Teams are also encouraged to run through [Supergraph Architecture Framework (SAF) assessment](https://saf.apollographql.com/) to better understand their GraphQL posture across several pillars like Reliability, Security, Operational Excellence, Performance and Developer Experience
```

### Mission and Vision

ACME is looking to establish a supergraph that is the preferred development platform for all services.

The goal of the ACME supergraph is to be the aggregation layer on top of the a domain-based services architecture, defining and mapping relationships in the data across bounded contexts. Key to our success will be to make the supergraph the preferred mode of development over REST APIs as measured by the percentage of requests going to our GraphQL instance vs their REST counterparts. 

The surface of the graph will span each of our current domain services, beginning with the most used domains first.

The success outcomes for the mission established by ACME are as follows:

1. We will simplify the API landscape by making the GraphQL supergraph the preferred platform over continued development of RESTful APIs. 
2. Our focus will be on building reusable Domain APIs to focus on reusability, maintainability and scalability while minimizing the timelines required to update business logic.

> 3. We will also establish a "GraphQL Shepherds" guild to play a key role in training and up-leveling GraphQL expertise across the dev teams.

Reviewer: Don't we typically try to shy away from "guilds" here?

Author: Fair - I think I'll just change this to a more generic term like "group"

## Getting Help

To request help with the ACME graph, please reach out in #graphql in Slack. 

The following is a proposed learning path for contributors not familiar with Apollo GraphQL.

### Learning Paths

> Apollo offers two certification courses through their Odyssey platform: [Associate](https://www.apollographql.com/tutorials/certifications/apollo-graph-associate) and [Professional](https://www.apollographql.com/tutorials/certifications/apollo-graph-professional). Contributors are recommended to complete at least the Associate certification in order to understand the basics of GraphQL and Federation.

Reviewer 2: We have the new API orchestration associate path that should be recommended first: https://www.apollographql.com/tutorials/certifications/apollo-orchestration-associate

Author: @R4ph-t is this a replacement for the Associate certification? to clarify - the goal here is to get a foundational understanding of graphQL in general and federation in particular. from my understanding the orchestration certification focusses more on REST connectors. if these are different i'd want to list them all out and describe the use case for each certification

> Apollo offers two certification courses through their Odyssey platform: [Associate](https://www.apollographql.com/tutorials/certifications/apollo-graph-associate) and [Professional](https://www.apollographql.com/tutorials/certifications/apollo-graph-professional). Contributors are recommended to complete at least the Associate certification in order to understand the basics of GraphQL and Federation.

Reviewer 4: ```suggestion
Apollo offers two certifications through their Odyssey platform: [Associate](https://www.apollographql.com/tutorials/certifications/apollo-graph-associate) and [Professional](https://www.apollographql.com/tutorials/certifications/apollo-graph-professional). Contributors are recommended to complete at least the Associate certification in order to understand the basics of GraphQL and Federation.
```

In addition to the certifications, Apollo also offers smaller modules for focusing on certain areas:

- [A bite-sized course on what federation is](https://www.apollographql.com/tutorials/introduction-to-federation)

> - [Basics of GraphOS with a strong focus on Studio usage](https://www.apollographql.com/tutorials/getting-started-with-graphos)

Reviewer 2: The Getting Started with graphOS series has been deprecated. 
@mabuyo can you please provide links to the best alternatives?

Reviewer 3: To future proof this, can we link to the Browse page (https://www.apollographql.com/tutorials/browse)? Then maybe call out the filters to let them explore what they can learn.

We already recommend the certifications above which involves a lot of courses, so I don't want to list out a whole bunch of other noise here.

- [Beta course for safe API delivery into GraphOS](https://www.apollographql.com/tutorials/graphos-shipping-supergraph)
- [Beta mini-course on growing your supergraph](https://www.apollographql.com/tutorials/graphos-growing-supergraph)
- [Beta course around leveraging Apollo Client on a React front-end](https://www.apollographql.com/tutorials/client-side-graphql-react)

### Enterprise Modules

Enterprise modules require ACME contributors to log into Odyssey using their SSO credentials
- [Effective graph stewardship](https://www.apollographql.com/tutorials/effective-stewardship)
- [Observability in GraphOS](https://www.apollographql.com/tutorials/supergraph-observability)
- [Testing for both clients and the servers](https://www.apollographql.com/tutorials/testing)

### Additional Resources

> - Another great reference for GraphQL is the official [GraphQL website](https://graphql.org/learn/)

Reviewer 3: We also have https://graphql.com/, which Apollo owns.

* Deprecations of fields or subgraphs ([see the below deprecations section](#deprecations))
* If a review would be helpful

#### How to Engage

While schema reviews are helpful, the core graph team also realizes that meeting after meeting isn't conducive to productivity. 

> With that, the SWG strongly recommends schema reviews using Apollo's [schema proposals](https://www.apollographql.com/docs/graphos/platform/schema-management/proposals) feature. Managing schema changes using schema proposals enables teams to propose schema changes that are validated against our development variant, running composition, linting and schema checks on every change.

Reviewer: Thank you for updating this- this doc is quite old, obviously. :)

Author: happy to!

To engage with the SWG, please reach out to #graphql-swg on Slack along with a link to the appropriate schema proposal. 

#### Next Steps

Once the SWG has received the proposal, they will begin to provide comments and suggestions within the tool. 

For new subgraphs, there will also be a separate meeting to discuss the proposal in-depth and provide further feedback. 

Once both the SWG and subgraph team(s) agree on the change, the proposal will be considered "approved" and will move to be implemented into the supergraph.

> Subgraph teams can pull down the approved proposal using `rover` and once published to the supergraph the proposal will be marked as "implemented"

Reviewer: Might be worth having an example here?

Author: What did you have in mind? I wan't to avoid duplicating instructions from the docs link here
